protostar/starknet/cheatable_cached_state.py

```diff
@@ -9,6 +9,10 @@
 
 from protostar.starknet.cheaters import BlockInfoCheater, Cheaters
 from protostar.starknet.types import AddressType, ClassHashType, SelectorType
+from protostar.starknet.data_transformer import CairoOrPythonData
+
+
+CallData = list[int]
```

Comment:
Can we use aliases defined in `protostar/starknet/data_transformer.py` instead?

Reply:
turned out this type's redundant [remove redundant type definition](https://github.com/software-mansion/protostar/pull/1060/commits/dbc97fa6c34a132d16a66a6b45cfd40728e00e6b)